script/enum-generator/requirements.txt

```diff
@@ -1,2 +1,2 @@
 wget==3.2
-beautifulsoup4==4.9.1
\ No newline at end of file
+beautifulsoup4==4.9.1
```

Comment:
:thinking:

Reply:
Oops, Vim automatically added a missing trailing line to this file when I saved it.
I will remove this change.